src/Input/Input.jsx

```diff
@@ -197,7 +213,10 @@ const StyledInput = styled.input(
   variant({ variants }),
   ({ tinted, error, warn, disabled }) => ({
     backgroundColor: tinted && !(error || warn || disabled) && 'white',
-    borderColor: tinted && !(error || warn || disabled) && 'white'
+    borderColor: tinted && !(error || warn || disabled) && 'white',
+    '::-webkit-search-cancel-button': {
```

Comment:
Maybe override the type to be text in this case so we can hide this close button on all browsers, not just webkit

Reply:
I verified this on 5 browsers, it works and hides the icon on all of them. This seems to be a good solution.

Reply:
Not on mozilla ... they do not implement webkit properties ... use `caniuse` for this kind of stuff, not random tests

https://caniuse.com/?search=webkit-search-cancel-button

Reply:
From what I read this should not be a problem for firefox since it does not implement the x icon by default and the developer is expected to style it by himself from scratch. Since we want to remove it, we have nothing to remove on firefox.

Reply:
https://stackoverflow.com/a/37846330